index.html

```diff
@@ -5,13 +5,13 @@
     <link rel="icon" type="image/png" href="/favicon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>
-      Small Business Lending Data Submission Platform | Consumer Financial
+      Small Business Lending Data Filing Platform | Consumer Financial
       Protection Bureau
     </title>
     <meta name="robots" content="noindex" />
     <meta
       name="description"
-      content="Use the CFPB small business lending data submission platform to upload your lending data, review validation results, and submit your filing."
+      content="Use the CFPB Small Business Lending Data Filing Platform to upload your lending data, review validation results, and submit your filing."
```

Comment:
@billhimmelsbach 
I think this is totally fine as a quick update. I will note it as one to double check though. 

I did a look of what we do elsewhere and I found this: 
Upload your loan application data, review validations, certify the accuracy and completeness of the data, and submit data for the filing year.

Reply:
I checked with Angel and he said this looks fine

Reply:
Great! 👍